paddle/fluid/framework/data_feed.cc

```diff
@@ -388,8 +419,14 @@ void InMemoryDataFeed<T>::GlobalShuffle() {
       }
     }
   }
-  for (int j = 0; j < send_vec.size(); ++j) {
-    if (send_vec[j].size() != 0) {
+  // shuffle the sequence of sending to avoid network timeout error
```

Comment:
so the previous timeout problem is fixed?

Reply:
yes,  this can solve the timeout problem～